src/NotepadNext/docks/FolderAsWorkspaceDock.ui

```diff
@@ -27,19 +27,77 @@
     <property name="bottomMargin">
      <number>0</number>
     </property>
+    <item>
+     <widget class="QMenu" name="menuFile">
```

Comment:
Can QtCreator create a QMenu in the ui file? or some other method (e.g. manually)? I'm just interested since I've never seen this done before :)

Reply:
It can't. I googled for it but at least Qt 5 can't. For me it looks like minor inconvenience everyone accustomed to.